### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Chethan64**
Date: 19/02/2024

Developer Name: Chethan Raghavendra

---

## Description

Updating README.md for better setup of the project
- Added steps to create the environment file and run/debug configuration
- Updated few unclear steps

### Documentation Updated?

- [ ] Yes
- [ ] No

### Under Feature Flag

- [ ] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [ ] No

### Breaking Changes

- [ ] Yes
- [ ] No

### Development Tested?

- [ ] Yes
- [ ] No

## Screenshots
NA

## Test Coverage
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,35 +52,50 @@ Then, we unzip it to the folder where we want Maven to live.
...
-1. `create database skilltree;`(semicolon is important here)
-2. `show databases;`
-3. `create user 'testuser' identified by 'testpassword';` (Username: testuser, Password: testpassword)
-4. `grant all on skilltree.* to testuser;`
+```
+CREATE DATABASE skilltree;
+SHOW DATABASES;
+CREATE USER 'testuser' IDENTIFIED BY 'testpassword';
+GRANT ALL ON skilltree.* TO testuser;
+```
```
Comment (satyam73) [inline README.md:90]: What's the need of this section if docker compose file is doing everything for us

File: README.md [hunk 2]
```diff
@@ -52,35 +52,50 @@ Then, we unzip it to the folder where we want Maven to live.
...
 
-## Steps for Creating the Database
+## Steps for setting up skill-tree in Intellij
 
 1. After creating the database project needs to be compiled.
 2. Open skill-tree-backend in intellij.
 3. Java_Home path needs to be added here.
-4. You can either add the existing path and jdk 17 can be downloaded inside intellij.
+4. You can either add the existing path and JDK 17 can be downloaded inside intellij.
+5. If adding from the existing path, go to Settings > Project Structure > Choose the earlier installed Java 17 SDK.
```
Comment (satyam73) [inline README.md:98]: Missing steps here, I can't see how will the other person load env file, also this points can be more clear

**Comment by satyam73** [comment]
Please attach necessary ticket for this PR

**Comment by satyam73** [comment]
Also fill the description properly please

File: README.md (hunk 1 above)
Comment (Chethan64) [reply README.md:90]: Docker compose file only executes the first command if I'm not wrong, I had to execute the rest of the commands

File: README.md (hunk 2 above)
Comment (Chethan64) [reply README.md:98]: I have updated with a new section having steps to create .env file and run configurations

**Comment by Chethan64** [comment]
> Please attach necessary ticket for this PR
Should I go through the same creation of issue and TCR workflow?

**Review by iamitprakash — CHANGES_REQUESTED** [review]
add missing details in desc section

**Comment by Chethan64** [comment]
Closing this PR and created a new one from Real-Dev-Squad:update-readme-setup to Real-Dev-Squad:develop

**Pull request closed without merging**